konfera/tests/test_views.py

```diff
@@ -81,6 +92,16 @@ def test_cfp_non_existing_event(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, 404)
 
+    def test_cfp_not_allowed(self):
+        url = reverse('event_cfp_form', kwargs={'slug': 'cfp-not-allowed'})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 404)
+
+    def test_cfp_after_deadline(self):
+        url = reverse('event_cfp_form', kwargs={'slug': 'passed-cfp'})
+        response = self.client.get(url)
+        self.assertIn('{% trans "Thank you for your interest, but unfortunately', str(response.content))
```

Comment:
`{% trans "` should not be in the output HTML file, if it's there it means something went wrong. (in this case you forgot the closing `%}` in the templates)